modules/op-updating-tekton-hub-with-custom-categories-and-catalogs.adoc

```diff
@@ -0,0 +1,66 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-hub-with-openshift-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="updating-tekton-hub-with-custom-categories-and-catalogs_{context}"]
+= Updating {tekton-hub} with custom categories and catalogs
+
+[role="_abstract"]
+Cluster administrators can update Tekton Hub with custom categories and catalogs that reflect the context of their organization. 
+
+[discrete]
+.Procedure
+
+. Edit the `categories` and `catalogs` fields in the Tekton Hub CR.
++
+.Example: Default categories and catalogs in the Tekton Hub CR 
```

Comment:
We don't specify any default categories, catalog, scopes and default scopes in the CR, by default data will be used from Hub API config map, if user specifies any data in the CR, those values will be used and not the default one's